### PR TITLE
Create Remapper

### PR DESCRIPTION
**Pull request by dmitryrck**
Apply the mappings when creating Outgos.

**WARNING**: It is still a work in progress, but reviews are welcome. The only thing left is to implement the _starts with_ and _ends with_ feature.

### CODE REVIEW

**Comment by dmitryrck** [comment]
Ready for review \o/.

**Review by rafaelrpbelo — APPROVED** [review]
Awesome!

File: app/services/remapper_helper.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module RemapperHelper
+  module_function
+
+  def valid?(movement: nil, mapping: nil)
+    return false if mapping.blank? || movement.blank?
+
+    current_content = movement.send(mapping.field_to_watch).to_s.downcase
```
[the remark this answers is not in the document]
Comment (dmitryrck) [reply app/services/remapper_helper.rb:7]: OMG!

THANK YOU!

**Pull request merged**